lms/djangoapps/courseware/courses.py

```diff
@@ -230,7 +230,7 @@ def get_course_about_section(request, course, section_key):
                         u"Error rendering course=%s, section_key=%s",
                         course, section_key
                     )
-            return html
+            return HTML(html)
```

Comment:
@nedbat Should this really move into `render_to_string()` called above?  Should we assume that all calls to `render_to_string()` return `HTML()`?